droidlet/perception/robot/handlers/label_propagate.py

```diff
@@ -2,21 +2,57 @@
 Copyright (c) Facebook, Inc. and its affiliates.
 """
 from .core import AbstractHandler
-from tokenize import String
 import numpy as np
 import os
-import sys
-if "/opt/ros/kinetic/lib/python2.7/dist-packages" in sys.path:
-    sys.path.remove("/opt/ros/kinetic/lib/python2.7/dist-packages")
 import cv2
 import json
-from copy import deepcopy as copy
-from IPython import embed
-import time
-from scipy.spatial.transform import Rotation
 import glob
-import argparse
 from droidlet.lowlevel.robot_mover_utils import transform_pose
+from numba import njit
+from math import ceil, floor
+
+def compute_uvone(height, width):
+    intrinsic_mat = np.array([[256, 0, 256], [0, 256, 256], [0, 0, 1]])
+    rot = np.array([[0.0, 0.0, 1.0], [-1.0, 0.0, 0.0], [0.0, -1.0, 0.0]])
+    trans = np.array([0, 0, 0.6])
```

Comment:
especially this one.  why .6?  the .6 should be a named constant itself

Reply:
These are just hard-coded intrinsic and extrinsic parameter values. Eventually, I want to move all the transformation parameters and their associated utilities to a consolidated place (across Hello and Locobot). Waiting to see @soumith's changes to figure out what the best place to do that would be. Will add a todo for now.

Reply:
Right, and because they are hard-coded consts, it imo is nicer that they appear at the header of the file, denoted as consts, with a short comment about what the const is.   (agree with having a consolidated file long term).

Reply:
Yep, definitely will do that.

Reply:
updated to constants. also self-assigned a task to streamline how we process the camera parameters https://github.com/facebookresearch/fairo/issues/901